### PR TITLE
how-to/upgrade:optimize TiDB 3.0 Upgrade Operations Guide

### PR DESCRIPTION
**Pull request by scsldb**
### What is changed, added or deleted? 

Optimize TiDB 3.0 Upgrade Operations Guide

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 
v3.0

### CODE REVIEW

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 1]
```diff
@@ -6,29 +6,23 @@ aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/','/docs-cn/v3.0/how-to/upg
 
 # TiDB 3.0 升级操作指南
 
-本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 RC 版本升级到 TiDB 3.0 版本。TiDB 3.0 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md) 以及[TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。
+本文档仅适用于从 TiDB 2.0.1 版本或者 TiDB 2.1 RC1 版本升级到 TiDB 3.0.x 版本。如果版本早于 TiDB 2.0.1 请选择如下两个方案：
```
Comment (liubo0127) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
本文档仅适用于从 TiDB 2.0.1 及以上版本升级到 TiDB 3.0.x 版本。如果版本早于 TiDB 2.0.1 请选择如下两个方案：
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 2]
```diff
@@ -6,29 +6,23 @@ aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/','/docs-cn/v3.0/how-to/upg
...
-- 从 2.0.6 之前的版本升级到 3.0 之前，需要确认集群中是否存在正在运行中的 DDL 操作，特别是耗时的 `Add Index` 操作，等 DDL 操作完成后再执行升级操作
-- 2.1 及之后版本启用了并行 DDL，早于 2.0.1 版本的集群，无法滚动升级到 3.0，可以选择下面两种方案：
-    - 停机升级，直接从早于 2.0.1 的 TiDB 版本升级到 3.0
-    - 先滚动升级到 2.0.1 或者之后的 2.0.x 版本，再滚动升级到 3.0 版本
+## 注意事项
 
-> **注意：**
->
-> 在升级的过程中不要执行 DDL 请求，否则可能会出现行为未定义的问题。
+1. TiDB 3.0.x 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md) 以及[TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。
```
Comment (liubo0127) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
1. TiDB 3.0.1 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md) 以及[TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。TiDB 3.0.2 及以上版本不再兼容 Kafka 版本的 TiDB Binlog
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 3]
```diff
@@ -43,58 +37,51 @@ Version: 0.9.0
...
     ```
 
-    如需变更，可参考 [如何调整进程监管方式从 supervise 到 systemd](/v3.0/how-to/deploy/orchestrated/ansible.md#如何调整进程监管方式从-supervise-到-systemd)，先使用备份 `/home/tidb/tidb-ansible-bak/` 分支变更进程监管方式再升级。
+## 修改配置文件
+
+>**注意:**
+>
+>* 请以 `tidb` 用户登录中控机。
+>
+>* TiDB Ansible 工作目录在 `/home/tidb/tidb-ansible` 目录。
```
Comment (liubo0127) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
>* TiDB Ansible 工作目录是 `/home/tidb/tidb-ansible`。
```

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 4]
```diff
@@ -43,58 +37,51 @@ Version: 0.9.0
...
+>
+>* 请以 `tidb` 用户登录中控机
+>
+>* TiDB Ansible 配置文件在 `/home/tidb/tidb-ansible` 目录
 
-- TiKV 配置中 `end-point-concurrency` 变更为 `high-concurrency`、`normal-concurrency` 和 `low-concurrency` 三个参数：
+### 修改 TiDB 集群组件配置文件
+
+1. 从备份的 Ansible 文件中拷贝集群的配置信息到新的配置文件，配置文件名称是`/home/tidb/tidb-ansible/conf`
+2. TiKV 配置中 `end-point-concurrency` 变更为 `high-concurrency`、`normal-concurrency` 和 `low-concurrency` 三个参数：
```
Comment (liubo0127) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
2. 如果 TiKV 原配置中设置了 `end-point-concurrency` ，则变更为 `high-concurrency`、`normal-concurrency` 和 `low-concurrency` 三个参数：
```

**Comment by TomShawn** [comment]
@scsldb When the content is ready, please also update v3.1 as well, thanks~

File: v3.0/how-to/upgrade/from-previous-version.md (hunk 1 above)
Comment (yikeke) [reply v3.0/how-to/upgrade/from-previous-version.md]: Please address comments, thanks! @scsldb

File: v3.0/how-to/upgrade/from-previous-version.md [hunk 5]
```diff
@@ -6,29 +6,23 @@ aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/','/docs-cn/v3.0/how-to/upg
...
-- 从 2.0.6 之前的版本升级到 3.0 之前，需要确认集群中是否存在正在运行中的 DDL 操作，特别是耗时的 `Add Index` 操作，等 DDL 操作完成后再执行升级操作
-- 2.1 及之后版本启用了并行 DDL，早于 2.0.1 版本的集群，无法滚动升级到 3.0，可以选择下面两种方案：
-    - 停机升级，直接从早于 2.0.1 的 TiDB 版本升级到 3.0
-    - 先滚动升级到 2.0.1 或者之后的 2.0.x 版本，再滚动升级到 3.0 版本
+## 注意事项
 
-> **注意：**
->
-> 在升级的过程中不要执行 DDL 请求，否则可能会出现行为未定义的问题。
+1. TiDB 3.0.x 版本和 master 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md) 以及 [TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。
```
Comment (yikeke) [inline v3.0/how-to/upgrade/from-previous-version.md]:
```suggestion
1. TiDB 3.0.x 版本和 master 版本兼容 [TiDB Binlog Cluster 版本](/v3.0/reference/tidb-binlog-overview.md)。TiDB 3.0.1 版本兼容 [Kafka 版本的 TiDB Binlog](/v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md)，TiDB 3.0.2 及以上版本不再兼容 Kafka 版本的 TiDB Binlog。
```
This comment is from @liubo0127 : https://github.com/pingcap/docs-cn/pull/1875#discussion_r326996973

**Comment by yikeke** [comment]
Closed for stale commits. Please open another PR to make the changes.

**Pull request closed without merging**